components/Image/Image.js

```diff
@@ -4,17 +4,21 @@ import { Image as ImageNative, View } from 'react-native';
 
 import Skeleton from '../Skeleton';
 import styles from './Image.style';
-import resizeWithAkamai from './modules/resizeWithAkamai';
+import resizeWithFastly from './modules/resizeWithFastly';
```

Comment:
if you don't use `resizeWIthAkamai` you should remove the module.

Reply:
It is still used in stories and other projects. I have just added it for jeffrey